csunplugged/tests/topics/TopicsTestDataGenerator.py

```diff
@@ -310,3 +312,35 @@ def add_challenge_lesson_relationship(self, challenge, lesson, set_number, numbe
             challenge_number=number,
         )
         relationship.save()
+
+    def create_glossary_term(self, number):
+        """Create glossary term object.
+
+        Args:
+            number: Identifier of the glossary term (int).
+
+        Returns:
+            GlossaryTerm object.
+        """
+        term = GlossaryTerm(
+            slug="term-{}".format(number),
+            term="Term {}".format(number),
+            definition="Defintion for term {}".format(number),
+        )
+        term.save()
+        return term
+
+    def add_lesson_resource_relationship(self, lesson, resource, number):
+        """Add relationship between challenge and lesson objects.
+
+        Args:
+            lesson (Lesson): Lesson to add relationship between.
+            resource (Resource): Resource to add relationship between.
+            number (int): Number to display in description.
```

Comment:
parameter types should be after the description rather than the parameter name

Reply:
After discussion with @hayleyavw, we agreed we prefer this way compared to at the end of the description as it has been done. Will move type over time.